request.el

```diff
@@ -927,16 +928,16 @@ Currently it is used only for testing.")
          &aux
          (cookie-jar (convert-standard-filename
                       (expand-file-name (request--curl-cookie-jar)))))
+  "BUG: Simultaneous requests are a known cause of cookie-jar corruption."
   (append
    (list request-curl "--silent" "--include"
          "--location"
-         ;; FIMXE: this way of using cookie might be problem when
-         ;;        running multiple requests.
          "--cookie" cookie-jar "--cookie-jar" cookie-jar
          "--write-out" request--curl-write-out-template)
    (when (plist-get (request--curl-capabilities) :compression)
      '("--compressed"))
    request-curl-options
+   (when (plist-get (request--curl-capabilities) :compression) (list "--compressed"))
```

Comment:
You may want to remove this or the two previous lines, else the `--compressed` flag will be added twice.

Reply:
Ah, thanks.